app/src/renderer/main.js

```diff
@@ -99,45 +93,23 @@ async function main() {
     next()
   })
 
-  ipcRenderer.on(`error`, (event, err) => {
-    switch (err.code) {
-      case `NO_NODES_AVAILABLE`:
-        store.commit(`setModalNoNodes`, true)
+  // wait until backend was reached to connect
```

Comment:
I would change the comment because we are not actually waiting (he's waiting internally) + if you want to move this to a separate function